src/book/mod.rs

```diff
@@ -25,8 +25,10 @@ pub struct MDBook {
     pub livereload: Option<String>,
 
     /// Should `mdbook build` create files referenced from SUMMARY.md if they
-    /// don't exist
-    pub create_missing: bool,
+    /// don't exist? If `None`, use the setting specified in `book.toml`,
+    /// otherwise use this.
+    /// This is for backward compatibility only.
+    pub create_missing: Option<bool>,
```

Comment:
You could probably get away with removing this completely. We're already changing the type from `bool` to `Option<bool>` so any 3rd parties using it will get compile errors. If that's the case, we may as well remove it completely so `create_missing` is only defined in one place (the `Build` struct under `Config`).

Reply:
Agreed, that's a better approach.  I missed the fact that the book config was available there!